kvmagent/kvmagent/plugins/host_model_mount_plugin.py

```
@@ -192,15 +484,31 @@ def mount_model_center(self, req):
             # Mount
-            success, error = mount_juicefs(cmd.storageUrl, mount_point)
+            success, error, did_mount = mount_juicefs(cmd.storageUrl, mount_point)
             if success:
+                # Only save registry entry if we actually performed a new mount.
+                # If the mount point was already mounted, we should NOT overwrite
+                # the registry entry because the existing mount may have a different
+                # storage URL than what we're trying to save.
+                if did_mount:
+                    try:
+                        _save_mount_registry_entry(cmd.modelCenterUuid, cmd.storageUrl, mount_point)
+                    except Exception as save_err:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)